net/picotls/Portfile

```diff
@@ -65,7 +73,7 @@ destroot {
     move ${cmake.build_dir}/cli ${destroot}${prefix}/bin/picotls_cli
     copy ${worksrcpath}/include/picotls ${destroot}${prefix}/include
     copy ${worksrcpath}/include/picotls.h ${destroot}${prefix}/include
-    foreach lib {libpicotls-core.a libpicotls-mbedtls.a libpicotls-minicrypto.a libpicotls-openssl.a} {
+    foreach lib {libpicotls-core.a libpicotls-fusion.a libpicotls-minicrypto.a libpicotls-openssl.a} {
```

Comment:
This does not work, `libpicotls-fusion.a` is not built on powerpc (or with gcc).